indico/modules/events/editing/schemas.py

```diff
@@ -138,6 +139,7 @@ class Meta:
     can_create_internal_comments = fields.Function(
         lambda editable, ctx: editable.contribution.event.can_manage(ctx.get('user'), permission='paper_editing')
     )
+    review_conditions_ok = fields.Function(lambda editable, ctx: editable.review_conditions_ok())
```

Comment:
Why not make the method on the Editable a property? That way you can use a simple `fields.Bool()` here